.circleci/config.yml

```diff
@@ -9468,7 +9468,7 @@ workflows:
           build_environment: pytorch-win-vs2019-cuda11-cudnn8-py3
           cuda_version: "11.3"
           name: pytorch_windows_vs2019_py36_cuda11.3_build
```

Comment:
s/py36/py38, and also other places for py36

Reply:
Good catch! Thank you

Reply:
didn't check closely, for example if we only have py38, maybe just py3 is better (if we do support other py3 versions, then nvm), so that future upgrades don't need to change the name again - it helps for some metrics to be stable with the test names.

Reply:
I updated the code so that there's just one place you have to change for the naming in `windows_build_definitions.py`. I think that makes it somewhat easier but it would still require updating for every new version